sdk/communication/Azure.Communication.Chat/src/ChatClient.cs

```diff
@@ -54,18 +54,17 @@ protected ChatClient()
         /// <summary>Creates a ChatThreadClient asynchronously. <see cref="ChatThreadClient"/>.</summary>
         /// <param name="topic">Topic for the chat thread</param>
         /// <param name="participants">Participants to be included in the chat thread</param>
+        /// <param name="repeatabilityRequestId"> If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-ID and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-ID is an opaque string representing a client-generated, globally unique for all time, identifier for the request. It is recommended to use version 4 (random) UUIDs. </param>
         /// <param name="cancellationToken">The cancellation token for the task.</param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "AZC0015:Unexpected client method return type.", Justification = "ChatThreadClient needs to be created by the ChatClient parent object")]
-        public virtual async Task<ChatThreadClient> CreateChatThreadAsync(string topic, IEnumerable<ChatParticipant> participants, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<CreateChatThreadResult>> CreateChatThreadAsync(string topic, IEnumerable<ChatParticipant> participants, string repeatabilityRequestId = null!, CancellationToken cancellationToken = default)
```

Comment:
```cs 
string repeatabilityRequestId = null!
```
It seems like this really can be null, so why do we do this? This ties into my other comment about libraries not generally having nullable refs enabled.

Reply:
Right, for these parameters, the signature should signify the intent, so I agree we should use the proper nullable declaration here :
string? repeatabilityRequestId = null instead of string repeatabilityRequestId = null!
Will do it for all the other methods I find.